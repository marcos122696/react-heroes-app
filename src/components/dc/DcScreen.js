import React from 'react'
import { HeroList } from '../hero/HeroList'

export const DcScreen = () => {
  return (
    <div>
        <h1 className="animate__animated animate__hinge">
            DcScreen
        </h1>
        <hr />
        
        <HeroList publisher='DC Comics'/>
    </div>
  )
}
