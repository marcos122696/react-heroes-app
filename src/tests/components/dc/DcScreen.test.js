import { shallow } from "enzyme";
import { DcScreen } from "../../../components/dc/DcScreen";
import React from 'react';

describe('Pruebas en en el componente <DcScreen />', () => {
    
    test('Debe de mostrarse correctamente', () => {

        const wrapper = shallow(<DcScreen />);
        expect(wrapper).toMatchSnapshot();
    });
});