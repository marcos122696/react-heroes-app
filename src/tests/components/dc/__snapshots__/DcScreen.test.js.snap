// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en en el componente <DcScreen /> Debe de mostrarse correctamente 1`] = `
<div>
  <h1
    className="animate__animated animate__hinge"
  >
    DcScreen
  </h1>
  <hr />
  <HeroList
    publisher="DC Comics"
  />
</div>
`;
